website/server/libs/inbox/index.js

```diff
@@ -45,3 +49,9 @@ export async function clearPMs (user) {
     Inbox.remove({ownerId: user._id}).exec(),
   ]);
 }
+
+export async function updateMessage (message) {
+  await Inbox.update({
```

Comment:
if the message is a mongoose model why not change the message object directly and then call `.save` on it? Instead of updating the entire object on the db. Note that in this case you'll have to mark the `flags` as modified with `message.markModified('flags')` when changing them

Reply:
oh thats even better^^, not that experienced with mongoose^^ thankfully we have reviews haha, changed to `.save`